.rubocop.yml

```diff
@@ -28,6 +28,16 @@ Metrics/LineLength:
   Max: 100
   Exclude:
     - Gemfile
-
+  
 Style/Documentation:
   Enabled: false
+
+
+Style/FormatString:
+  Enabled: false
+Style/DotPosition:
+  Enabled: false
+Style/TrailingWhitespace:
+  Enabled: false
```

Comment:
Fair enough. I've approved, so feel free to merge.

You know there's a setting in your editor for that whitespace thing though ;-)